processor/metricstransformprocessor/metrics_transform_processor.go

```diff
@@ -124,3 +126,27 @@ func (mtp *metricsTransformProcessor) updateLabelOp(metric *metricspb.Metric, op
 		// label value update
 	}
 }
+
+func (mtp *metricsTransformProcessor) changeDataType(metric *metricspb.Metric, op Operation) {
```

Comment:
It seems like the parameter `op` is never used here. I am not sure what the best practice for this is, but maybe we can just remove this parameter here since it is never used?